third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -1815,7 +1727,7 @@ pub(crate) struct Module {
     function_map: HashMap<Identifier, usize>,
     // struct name to index into the Loader type list
     // This allows a direct access from struct name to `Struct`
-    struct_map: HashMap<Identifier, CachedStructIndex>,
+    struct_map: HashMap<Identifier, usize>,
```

Comment:
why can't we use this usize instead of the struct name in StructName

Reply:
the problem is for comparision: the struct can have reference in other module so this index is not a globally applicable index, thus if we use usize we can no longer use it for comparision (in fact we are moving away from the global index in this PR)